pkg/external_repos/introspect.go

```diff
@@ -153,6 +153,10 @@ func IntrospectAll(force bool) (int64, []error) {
 			errors = append(errors, err)
 		}
 	}
+	err = repoDao.OrphanCleanup()
```

Comment:
What about a small comment here to say that repository orphan cleanup is called here because it's a convenient place for it, but not because it's actually needed for introspection?

Reply:
actually, the more I think about it the more I think it fits here. maybe no explanation necessary!

Reply:
the _more_ I think about this, I realize the same thing could be said for `rpm.OrphanCleanup()`. We just call these in `IntrospectAll()` because the cron job will trigger them, right? `IntrospectURL()` on the other hand does not call `OrphanCleanup()`.

Anyway, I think this is fine. I had just assumed `rpm.OrphanCleanup()` was more directly tied to introspection specifically.

Reply:
sounds good.  Yes, i could see moving those two calls to a cleanup method might make it more clear, but maybe not necessary at this point